### PR TITLE
New SFA, FSF ship and detailed shiplore.

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
What it says up there. Also gives the SFA ship an extra marine. Credits to Robotic Potato for interior design

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Comment by Caelphon** [comment]
Danse approached me, and I gave my authorisation for these to be merged if there are any worries.

File: maps/away/ships/sol_ship_ghostroles.dm [hunk 1]
```diff
@@ -83,7 +83,7 @@
 	mob_name_prefix = "Pfc. "
 
 	spawnpoints = list("sfa_navy_crewman")
-	max_count = 1
+	max_count = 2
```
Comment (NonQueueingMatt) [inline maps/away/ships/sol_ship_ghostroles.dm:86]: Undocumented.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: maps/away/ships/sol_ship_ghostroles.dm (hunk 1 above)
Comment (TheDanseMacabre) [reply maps/away/ships/sol_ship_ghostroles.dm:86]: Slipped my mind

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**